src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size() 메소드로 Set의 크기를 확인")
+    void setSize() {
+        assertThat(numbers).hasSize(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("Set의 contains() 메소드로 1,2,3의 값이 존재하는지 확인")
+    void setContains(int values) {
```

Comment:
매개변수가 `@ValueSource`로부터 전달받는 값 하나를 의미하니 이름을 단수형으로 지어보면 어떨까요?
```suggestion
    void setContains(int value) {
```